### DISCUSSION TITLE
How to stop a background running pipeline?

### DISCUSSION THREAD

**Discussion by 1riyu** [nextflow-io/nextflow#2924] [Q&A] [answered]
Hi,

I ran a TransPi pipeline with:
"nextflow -bg run TransPi.nf --all --reads '/home/geninfo/kcapel/Cassiano/AC567/AC567_*_R[1,2]*.fastq.gz' \
     --k 35,55,75,85 --maxReadLen 100 --outdir Results_AC567 -with-tower -profile conda --myConda"

But it failed due to insufficient memory and, for some reason, when I access the nextflow tower it shows that the pipeline is still running.
I wanna rerun the pipeline but first I need to stop this current one.
I've tried using the kill command in two ways:

_(base) kcapel@seal:~/TransPi$ ps -ef | grep nextflow
kcapel   49050 46822  0 16:08 pts/16   00:00:00 grep --color=auto nextflow
(base) kcapel@seal:~/TransPi$ kill 46822_

and 

_(base) kcapel@seal:~/TransPi$ more .nextflow.pid 
(base) kcapel@seal:~/TransPi$ kill 8676_

but none of them seems to work as it is still running in the tower.

Can anyone help?

**Comment by lbrandhoff-w** [comment, marked as answer]
Have you tried killing it with `kill -9 8676`?
I also experienced what you described and this worked for me!

**Reply by 1riyu** [reply]
Thank you for the answer. Aparently it was some sort of problem with my nextflow tower, the kill command worked but for some reason it kept showing that the pipeline was still running.
